lib/app.rb

```diff
@@ -3,30 +3,36 @@
 require_relative './player.rb'
 require_relative './input_handler.rb'
 require_relative './constants.rb'
-require_relative './parser.rb'
 
 class App
-    def initialize(args={})
-        @game = args.fetch(:game, new_game)
-        @view = args.fetch(:view, View.new)
-        @parser = args.fetch(:parser, Parser.new)
-        @input_reader = args.fetch(:input_reader, InputHandler.new(IO.new(1)))
-    end
+  def initialize(args = {})
+    @game = args.fetch(:game, new_game)
+    @view = args.fetch(:view, View.new)
+    @delegator = args.fetch(:delegator)
+    @handler_table = args.fetch(:handler_table)
+    @input_reader = args.fetch(:input_reader, InputHandler.new(IO.new(1)))
+  end
 
-    def new_game
-        return Game.new({ players: [Player.new("player1", "X"), Player.new("player2", "O")] })
-    end
+  def new_game
+    Game.new(players: [Player.new('player1', 'X'), Player.new('player2', 'O')])
+  end
 
-    def run
-        game_players = [Player.new("Player1", "X"), Player.new("Player2", "O")]
-        game_args = { players: game_players }
-        @game = Game.new(game_args)
+  def run
+    game_players = [Player.new('Player1', 'X'), Player.new('Player2', 'O')]
```

Comment:
why are the capitalizations for player different between this line and new_game? Can these defaults be put into constants or have the list be extracted out?